gneiss/plot/_dendrogram.py

```diff
@@ -0,0 +1,235 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the GPLv3 License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+"""Drawing trees.
+
+Draws horizontal trees where the vertical spacing between taxa is
+constant.  Since dy is fixed dendrograms can be either:
```

Comment:
dx and dy refer to the spacing between the points within the tree layout.  This documentation was also directly ported pycogent.  Probably should ditch that since it is dated ...